ixofoundation/ixo-webclient
# Ixo 273 split out the styles

## Scope
Split out styles for Projects

## Work Done
Split out styles for
- ProjectsContainer
- ProjectsCard
- ProjectsDashboard
- ProjectsHero

into

- ProjectsContainer.styles.ts
- ProjectsCard.styles.ts
- ProjectsDashboard.styles.ts
- ProjectsHero.styles.ts

## Steps to Test
View styling within projects scope
See styling moved out of above files and into their respective style files

## Screenshots
![image](https://user-images.githubusercontent.com/35291291/78567903-a70b2200-7821-11ea-951b-4e6cc92a99a0.png)
